testsuite/run_forge.sh

```diff
@@ -27,7 +27,7 @@ fi
 
 # available clusters to choose from
 # forge-1 is used for continuous testing exclusively
-FORGE_CLUSTERS=("aptos-forge-0" "aptos-forge-2")
+FORGE_CLUSTERS=("aptos-forge-big-0" "aptos-forge-big-1" "aptos-forge-0")
```

Comment:
Is this used only for land blocking?

Reply:
So here's the state of things:
* land-blocking tests will use the above cluster selection, from a list of hardcoded clusters
* continuous tests have a single hard-coded cluster. These are `aptos-forge-1` and `aptos-forge-2`: https://github.com/aptos-labs/aptos-core/blob/main/.github/workflows/continuous-e2e-tests.yaml
* pre-release tests are using the new python wrapper, and thus could schedule on any `aptos-forge-*` cluster: https://github.com/aptos-labs/aptos-core/blob/main/.github/workflows/pre-release-e2e-tests.yaml 
* the new shadow workflow does the same: https://github.com/aptos-labs/aptos-core/blob/main/.github/workflows/shadow-e2e-tests.yaml

Reply:
So tl;dr after this PR lands, these new clusters are available for testing for land-blocking, shadow, and pre-release. @perryjrandall we should probably have a way to only have land-blocking tests on one cluster tho

Reply:
Yeah we just need a config somewhere, we could store in repo or store on the cluster somehow itself?